closed_issues.rb

```diff
@@ -1,12 +1,14 @@
-MILESTONE = 36
-ACCESS_TOKEN = "your github access token"
+MILESTONE = 47
+# subtract 6 from actual Sprint milestone number for the ManageIQ/manageiq repo
+ACCESS_TOKEN = "your access token"
 
 require_relative 'sprint_statistics'
 prs = SprintStatistics.new(ACCESS_TOKEN).pull_requests("ManageIQ/manageiq", :milestone => MILESTONE, :state => "closed")
 
-puts "Milestone Statistics for: #{prs.first.milestone.title}"
-puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS"
-puts "--------------------------------------------------"
-prs.each do |i|
-  puts "#{i.number},#{i.title},#{i.user.login},#{i.assignee && i.assignee.login},#{i.labels.collect(&:name).join(" ")}"
-end
+open('closed_issues_master_repo.csv', 'w') { |f|
```

Comment:
```ruby
File.open('closed_issues_master_repo.csv', 'w') do |f|
  # stuff
end
````
rather than
```ruby
File.open('closed_issues_master_repo.csv', 'w') { |f|
  # stuff
}
````